src/store/env-context.js

```diff
@@ -0,0 +1,15 @@
+import { createContext, useContext } from "react";
+
+const EnvContext = createContext({
+  musicApi: "",
+});
+
+export const EnvContextProvider = (props) => {
+  return <EnvContext.Provider value={{
+    // TODO: store urls in env files.
+    musicApi: "https://localhost:7223/api"
+  }}>{props.children}</EnvContext.Provider>;
+};
```

Comment:
1. Is it a good idea to place API urls in env files?
2. This context is gonna provide access to env variable (at the moment `musicApi` is hardcoded, but it'll be taken from env variables). Is it a good idea to have such context?